src/target/xtensa/xtensa.c

```diff
@@ -720,14 +787,19 @@ static inline bool xtensa_is_stopped(struct target *target)
 int xtensa_examine(struct target *target)
 {
 	struct xtensa *xtensa = target_to_xtensa(target);
-	unsigned int cmd = PWRCTL_DEBUGWAKEUP | PWRCTL_MEMWAKEUP | PWRCTL_COREWAKEUP;
+	unsigned int cmd = PWRCTL_DEBUGWAKEUP(xtensa) | PWRCTL_MEMWAKEUP(xtensa) | PWRCTL_COREWAKEUP(xtensa);
 
 	LOG_TARGET_DEBUG(target, "examine");
```

Comment:
We are mostly using LOG_TARGET_XXX in our fork. I'll create a patch

Reply:
I'm not attached to one or the other, so if using that makes maintaining your branch easier, I'm fine with you pushing it upstream.

Reply:
https://review.openocd.org/c/openocd/+/7232